anubis-management-api/src/tenants/operations.py

```diff
@@ -63,10 +63,19 @@ def get_service_paths(db: Session, skip: int = 0, limit: int = 100):
 def get_tenant_service_paths(
         db: Session,
         tenant_id: str,
+        name: str = None,
+        startswith: str = None,
         skip: int = 0,
         limit: int = 100):
-    return db.query(models.ServicePath).filter(
-        models.ServicePath.tenant_id == tenant_id).offset(skip).limit(limit).all()
+    db_service_paths = db.query(models.ServicePath).filter(
+        models.ServicePath.tenant_id == tenant_id)
+    if name:
```

Comment:
what if both name and startswith? if thing it gets complicated, my idea was to have just the `startswith`:)

Reply:
(and use name variable for it would be fine)